examples/lock-app/efr32/src/EventHandler.cpp

```diff
@@ -0,0 +1,204 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "EventHandler.h"
+
+#if defined(ENABLE_CHIP_SHELL)
+#include "AppTask.h"
+#include "lib/shell/Engine.h"
+#include "lib/shell/commands/Help.h"
+
+#define ENDPOINT_1 1
```

Comment:
This is odd ... like `#define FOO_123 123` is just a hardcoded number with extra steps.

Give this a name without the actual value. Also use constexpr insteac of C-style defines.

Reply:
changed to constexpr uint8_t LOCK_ENDPOINT 1